cmd/system-probe/modules/all_unsupported.go

```diff
@@ -6,3 +6,7 @@ import "github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
 
 // All System Probe modules should register their factories here
 var All = []module.Factory{}
+
+func inactivityEventLog(duration string) {
```

Comment:
This isn't necessary since the network tracer is only built on `windows` and `linux`.